front-end/app/styles/app.scss

```diff
@@ -18,88 +25,80 @@
-@import "ember-appuniversum/c-pill";
-@import "ember-appuniversum/c-main-container";
-@import "ember-appuniversum/c-main-header";
-@import "ember-appuniversum/c-main-footer";
-@import "ember-appuniversum/c-modal";
-@import "ember-appuniversum/c-sidebar";
-@import "ember-appuniversum/c-sidebar-action";
-@import "ember-appuniversum/c-textarea";
-@import "ember-appuniversum/c-toolbar";
-@import "ember-appuniversum/c-toggle-switch";
-
+@import 'ember-appuniversum/c-alert';
+@import 'ember-appuniversum/c-badge';
+@import 'ember-appuniversum/c-body-container';
+@import 'ember-appuniversum/c-brand';
+@import 'ember-appuniversum/c-button';
+@import 'ember-appuniversum/c-button-group';
+@import 'ember-appuniversum/c-card';
+@import 'ember-appuniversum/c-content';
+@import 'ember-appuniversum/c-content-header';
+@import 'ember-appuniversum/c-control';
+@import 'ember-appuniversum/c-data-table';
+@import 'ember-appuniversum/c-dropdown';
+@import 'ember-appuniversum/c-form';
+@import 'ember-appuniversum/c-heading';
+@import 'ember-appuniversum/c-help-text';
+@import 'ember-appuniversum/c-hr';
+@import 'ember-appuniversum/c-icon';
+@import 'ember-appuniversum/c-input';
+@import 'ember-appuniversum/c-label';
+@import 'ember-appuniversum/c-link';
+@import 'ember-appuniversum/c-list-horizontal';
+@import 'ember-appuniversum/c-list-navigation';
+@import 'ember-appuniversum/c-loader';
+@import 'ember-appuniversum/c-pagination';
+@import 'ember-appuniversum/c-pill';
+@import 'ember-appuniversum/c-main-container';
+@import 'ember-appuniversum/c-main-header';
+@import 'ember-appuniversum/c-main-footer';
+@import 'ember-appuniversum/c-modal';
+@import 'ember-appuniversum/c-sidebar';
+@import 'ember-appuniversum/c-sidebar-action';
+@import 'ember-appuniversum/c-textarea';
+@import 'ember-appuniversum/c-toolbar';
+@import 'ember-appuniversum/c-toggle-switch';
 
 // PLUGINS
-@import "ember-appuniversum/p-ember-power-select";
-@import "ember-appuniversum/p-duet-datepicker";
-
+@import 'ember-appuniversum/p-ember-power-select';
+@import 'ember-appuniversum/p-duet-datepicker';
 
 // UTILITIES
-@import "appuniversum/u-align-text";
-@import "appuniversum/u-break-word";
-@import "appuniversum/u-headings";
-@import "appuniversum/u-hide";
-@import "appuniversum/u-font-family";
-@import "appuniversum/u-font-weights";
-@import "appuniversum/u-paragraphs";
-@import "appuniversum/u-print";
-@import "appuniversum/u-responsive-spacings";
-@import "appuniversum/u-spacings";
-@import "appuniversum/u-visible";
-@import "appuniversum/u-widths";
+@import 'appuniversum/u-align-text';
+@import 'appuniversum/u-break-word';
+@import 'appuniversum/u-headings';
+@import 'appuniversum/u-hide';
+@import 'appuniversum/u-font-family';
+@import 'appuniversum/u-font-weights';
+@import 'appuniversum/u-paragraphs';
+@import 'appuniversum/u-print';
+@import 'appuniversum/u-responsive-spacings';
+@import 'appuniversum/u-spacings';
+@import 'appuniversum/u-visible';
+@import 'appuniversum/u-widths';
```

Comment:
Just for this whole file: It might be best to move all the appuniversum stuff to a separate file, so that we keep the SCSS file clean.

_You could make an issue of that in the Notion..._